patches/react-native-share-menu+6.0.0.patch

```diff
@@ -112,7 +112,16 @@ index e290cce..1c114ff 100644
          if error != nil {
              let exception = NSError(
                  domain: Bundle.main.bundleIdentifier!,
-@@ -130,7 +137,7 @@ public class ShareMenuReactView: NSObject {
+@@ -55,6 +62,8 @@ public class ShareMenuReactView: NSObject {
+         }
+ 
+         viewDelegate.openApp()
++        
++        ShareMenuReactView.detachViewDelegate()
```

Comment:
I thought we already had this in native-share, did something change?

Ah yeah, found it here, looks like different timing for the two use cases: https://github.com/infinitered/ExpensifyApp/pull/28/files#diff-1aa046d161a1081bd541198ac073c301a3773ce2f3591f5f88a5a6fc85c02e36R110

Reply:
Yeah from that issue you sent me the other day I think we might just be able to change the lifecycle method where that is currently getting called. Using `viewWillDisappear` instead [here](https://github.com/Expensify/react-native-share-menu/blob/master/ios/ReactShareViewController.swift#L52C17-L52C33) might be all we need.